db/migrate/20160203102911_migrate_users_to_have_role.rb

```diff
@@ -0,0 +1,23 @@
+class MigrateUsersToHaveRole < ActiveRecord::Migration
+  def up
+    Comfy::Cms::User.all.each do |user|
+      user.update_attribute(:role, new_role_for(user))
+    end
+  end
+
+  def down
+    Comfy::Cms::User.all.each do |user|
+      user.update_attribute(:admin, user.admin?)
+    end
+  end
+
+  private
+
+  def new_role_for(user)
+    if user.read_attribute(:admin)
+      Comfy::Cms::User.roles[:admin]
+    else
+      Comfy::Cms::User.roles[:user]
+    end
+  end
+end
```

Comment:
Ok, I said what I said because of the risks of changing data with difficult to rollback if something go wrong. But since the CMS users data is small, and in this case I would say that it is better do what you feel that is good to do it. I will give the thumbs, for it and leave you guys decide what is best.